FIAT/expansions.py

```diff
@@ -109,6 +110,37 @@ def xi_tetrahedron(eta):
     return xi1, xi2, xi3
 
 
+class PointExpansionSet(object):
+    """Evaluates the point basis on a point reference element."""
+
+    def __init__(self, ref_el):
+        if ref_el.get_spatial_dimension() != 0:
+            raise Exception("Must have a point")
+        self.ref_el = ref_el
+        self.base_ref_el = reference_element.Point()
```

Comment:
Is this guaranteed? Isn't it part of the public API?

Reply:
It's hard to tell where the boundaries of FIAT's public API lie.  Is it used anywhere from FEniCS or Firedrake?  If not, that it should not be seen as public API.

Reply:
Well on that basis, this is something that should be blanket changed everywhere to avoid inconsistency and should, therefore, be part of a separate PR to allow relevant discussion. @miklos1 can you confirm that this is definitely not used in FEniCS? I am not an expert on where I should be looking within the FEniCS project to confirm.

Reply:
I think the reason for `base_ref_el` is that the polynomials are constructed on elements that stretch the [-1; 1] interval and similar elements, meanwhile UFC elements stretch [0; 1], so an affine transformation is required to get the correct tabulations. But this difference makes no sense for the point, which has no coordinates.

I doubt it is used anywhere else, and I think a simple `git grep` on FFC, DOLFIN, FFC-X, DOLFIN-X should be able to confirm that.

Feel free to remove `self.base_ref_el` for other expansion sets, but please do that in a separate pull request.